src/widgets/resourceList.js

```diff
@@ -0,0 +1,65 @@
+import { Op } from 'sequelize';
+import { ActivityReport, ActivityReportObjective, ActivityReportObjectiveResource } from '../models';
+import { REPORT_STATUSES } from '../constants';
+
+export default async function resourceList(scopes) {
+  // Query Database for all Resources within the scope.
+  const res = await ActivityReport.findAll({
+    attributes: [
+      'id',
+    ],
+    where: {
+      [Op.and]: [
+        scopes.activityReport,
+        { calculatedStatus: REPORT_STATUSES.APPROVED },
+      ],
+    },
+    include: [
+      {
+        model: ActivityReportObjective,
+        as: 'activityReportObjectives',
+        include: [
+          {
+            model: ActivityReportObjectiveResource,
+            as: 'activityReportObjectiveResources',
+            required: false,
+          },
```

Comment:
Should this be ```required: true```?

Reply:
Yea good call was thinking statistics when we need to count no resources.